tools/cbmc/proofs/HTTP/IotHttpsClient_Connect/Makefile.json

```diff
@@ -1,5 +1,10 @@
 {
   "ENTRY": "IotHttpsClient_Connect",
+
+  # This is the length of the user data after the header in a buffer.
+  # This can be set to any size.
+  "USER_DATA_SIZE": "1000",
```

Comment:
There's no real need to set a variable up here if you're only using it in one place, I suggest erasing this line and writing `"USER_DATA_SIZE"="1000"` in the `"DEF"` array below.

Reply:
I prefer to leave this as it is.  This is a clean way to make it up front and center that there is an assumption being made.